addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/json/UserInfo.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.internal.json;
+
+/**
+ *
```

Comment:
According to the code guidelines all class-es should have a description of there meaning and goal above the author tag.